website/content/docs/concepts/variables.mdx

```diff
@@ -194,3 +234,5 @@ See [Workload Associated ACL Policies] for more details.
 [workload identity]: /nomad/docs/concepts/workload-identity
 [Workload Associated ACL Policies]: /nomad/docs/concepts/workload-identity#workload-associated-acl-policies
 [ACL policy namespace rules]: /nomad/docs/other-specifications/acl-policy#namespace-rules
+[The Chubby Lock Service for Loosely-Coupled Distributed Systems]: https://research.google/pubs/pub27897/
+[`nomad var lock`]: /nomad/docs/commands/var
```

Comment:
couple things:

* this links to `commands/var`, not `commands/var/lock` directly, and `commands/var` has a list of the sub-commands, but `lock` is not among them yet

* if the intent is to show implemented usage of the golang lock-lease-function-protector, then that commands/var doc doesn't really get me there.  maybe instead link to github instead?  right now it could be https://github.com/hashicorp/nomad/blob/f-gh-17449/command/var_lock.go#L240 -- but once we're sure of the release, it might be `https://github.com/hashicorp/nomad/blob/release/1.7.0/command/var_lock.go#L240`